### DISCUSSION TITLE
v2/zh/guide/markdown/mermaid

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#358] [Announcements]
# v2/zh/guide/markdown/mermaid

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/guide/markdown/mermaid.html

**Comment by Ayusummer** [comment]
启用了 Mermaid 之后无法正确加载, 这种情况应该如何处理呢?

![image](https://user-images.githubusercontent.com/59549826/200620151-3070a542-fdce-4131-bf02-3e7074d82f2e.png)

![image](https://user-images.githubusercontent.com/59549826/200620225-28402c42-8499-46d5-9466-cbb2e7b1719d.png)

![image](https://user-images.githubusercontent.com/59549826/200620682-546cf699-270f-4c77-b579-a53bf5383241.png)

**Reply by Mister-Hope** [reply]
9.2.0 引入了一些内部改动。最新版本应该是没有问题的

**Reply by Ayusummer** [reply]
9.2.0 是哪个版本呀, 现在最新版本不是 2.0.0-beta.123 么?

**Reply by Mister-Hope** [reply]
> 9.2.0 是哪个版本呀, 现在最新版本不是 2.0.0-beta.123 么?

9.2.0 指的是 mermaid 版本

**Reply by Ayusummer** [reply]
哦哦, 原来是这样, 我是直接 `pnpm intsall vuepress-theme-hope@next` 装的主题, 没有单独安装从依赖, 原来是需要再装下 mermaid 是么?(⊙x⊙;)

**Reply by Mister-Hope** [reply]
Mermaid 是插件依赖，它在前段时间在 9.2.0 进行了一些内部更改，导致 vite 无法处理。

**Reply by Ayusummer** [reply]
非常感谢, 看到您在 2.0.0-beta.122 版本修复了 mermaid 相关问题, 升级到该版本后可以正常展示 mermaid 图表了

![image](https://user-images.githubusercontent.com/59549826/202733224-aed41b72-86cf-4e8a-8d4c-8331fe6928fb.png)
